tests/components/cover/test_init.py

```diff
@@ -0,0 +1,12 @@
+"""The tests for Cover."""
+import homeassistant.components.cover as cover
+
+
+def test_deprecated_base_class(caplog):
+    """Test deprecated base class."""
+
+    class CustomCover(cover.CoverDevice):
+        pass
+
+    CustomCover()
+    assert "CoverDevice is deprecated, modify CustomCover" in caplog.text
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5418277)